components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -37,7 +76,7 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  //  self.recordMode = YES;
+  self.recordMode = YES;
```

Comment:
Don't forget to comment back out.

Reply:
Ah...Thanks for catching it!! I forgot it. Addressed in d93481634003f2329f773256e9aef6ce9df9e3ae.